### PR TITLE
add support for asdf

### PR DESCRIPTION
**Pull request by 5fff**
Updated the doctor script to support checking asdf as an alternative ruby version manager.

### CODE REVIEW

**Comment by rileykarson** [comment]
fyi @ScottSuarez

File: scripts/doctor [hunk 1]
```diff
@@ -29,18 +29,18 @@ warn() {
...
+    found=$(which -a ${1} | uniq | wc -l)
 }
 
 
-count_in_path "rbenv"
+count_in_path "rbenv asdf"
 if [ $found -eq 0 ]; then
     exitcode=1
-    warn "yel" "   rbenv not found. See https://github.com/rbenv/rbenv for information on installing it."
+    warn "yel" "   rbenv or asdf not found. See https://github.com/rbenv/rbenv for information on installing it."
```
Comment (roaks3) [inline scripts/doctor:39]: Might as well add both here for clarity. Maybe `Could not find rbenv or asdf. See https://github.com/rbenv/rbenv for information on installing rbenv (recommended), or https://asdf-vm.com/guide/getting-started.html for information on installing asdf.`

File: scripts/doctor [hunk 2]
```diff
@@ -29,18 +29,18 @@ warn() {
...
 else
     warn "grn" "   found!"
     echo "Checking ruby version..."
-    rbenv version
+    rbenv version || asdf current ruby
```
Comment (roaks3) [inline scripts/doctor:43]: Given the purpose of the doctor command, I feel like we should include some more information here that might help someone with problems. Specifically, could we surface which tool is being used to produce the ruby version?

One option here would be to refactor this into 2 blocks, one for `rbenv` and one for `asdf`, and ultimately only fail if neither is available.

**Review by roaks3 — COMMENTED** [review]
Overall makes sense and appears to function as expected, but since this is a `doctor` command meant to help diagnose issues, I think we should try to provide a little more info.

**Comment by 5fff** [comment]
@roaks3 thoughts?

**Comment by roaks3** [comment]
@5fff did you see my other comment about providing more information to the user?

**Comment by roaks3** [comment]
@5fff yes, I saw that you resolved that suggestion. I also made this comment, which is not addressed by the error message:

> Given the purpose of the doctor command, I feel like we should include some more information here that might help someone with problems. Specifically, could we surface which tool is being used to produce the ruby version?
>
> One option here would be to refactor this into 2 blocks, one for rbenv and one for asdf, and ultimately only fail if neither is available.

**Comment by rileykarson** [comment]
Closing as obsolete- we check specific Ruby versions and not for `rbenv` now.

**Pull request closed without merging**